performance/profiles/profile_spherical_expansion.cc

```diff
@@ -1,5 +1,5 @@
 /**
- * file   spherical_expansion_profile.cc
+ * file   profile_spherical_expansion.cc
```

Comment:
should be `@file ...` instead of `file ...`

Reply:
changed